Samples/BlazorExample/BlazorExample/DataAccess/PersonDal.cs

```diff
@@ -1,74 +1,74 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿//using System;
```

Comment:
Shouldn't this file just be deleted/removed - no sense leaving it here commented out.

Reply:
Sure I will correct this .